packages/teleterm/src/ui/DocumentGateway/DocumentGateway.tsx

```diff
@@ -24,16 +24,20 @@ import {
   Link,
```

Comment:
(Adding a comment here just to create a separate GH discussion)

I have a commit on my branch (that I rebased on master) where the types seem to be out of order:

https://github.com/gravitational/webapps/blob/04de0d9edd43bf25c6870a717b00275f10200f4c/packages/teleterm/src/ui/DocumentGateway/OfflineDocumentGateway.tsx#L1-L10 

I suppose `design` is treated like a package but `design/Alert` is treated as another group.

`teleterm/ui/components/LinearProgress` is put into a completely different group than `design/Alert` and `shared/components/Validation` which I guess makes sense as this file is within `packages/teleterm`.

I think top-level imports from `design` and our other packages should be treated the same way as imports from `design/Alert`, no?

Reply:
`design` and `design/Alert` not being together is another result of using `always-and-inside-groups`. It's just what autofix did to the existing code, if I remove all newlines and run it again then the imports look like this:

```typescript
import React, { useState } from 'react';
import { ButtonPrimary, Flex, Text } from 'design';
import * as Alerts from 'design/Alert';
import Validation from 'shared/components/Validation';

import LinearProgress from 'teleterm/ui/components/LinearProgress';

import { PortFieldInput } from './common';
import { DocumentGatewayProps } from './DocumentGateway';
```

I think we could configure the rule to treat our packages as the internal group, so that the final imports would look like this:

```
import React, { useState } from 'react';

import { ButtonPrimary, Flex, Text } from 'design';
import * as Alerts from 'design/Alert';
import Validation from 'shared/components/Validation';

import LinearProgress from 'teleterm/ui/components/LinearProgress';

import { PortFieldInput } from './common';
import { DocumentGatewayProps } from './DocumentGateway';
```

What do you think?